src/MinkowskiSum.jl

```diff
@@ -298,7 +298,11 @@ The support function of the Minkowski sum of sets is the sum of the support
 functions of each set. 
 """
 function ρ(d::AbstractVector{N}, msa::MinkowskiSumArray{N}) where {N<:Real}
-    return sum([ρ(d, Xi) for Xi in msa.array])
+    res = zero(N)
+    @inbounds for Xi in msa.array
+        res = res + ρ(d, Xi)::N
```

Comment:
I do not mind, but since when do you add type annotations? Note that this might actually be slower, as it has to check for the type.